resources/qml/Toolbar.qml

```diff
@@ -4,10 +4,10 @@
 import QtQuick 2.2
 import QtQuick.Controls 2.3
 
-import UM 1.2 as UM
+import UM 1.4 as UM
 import Cura 1.0 as Cura
 
-Item
+UM.UnclickableItem
```

Comment:
Is this change really needed? The item is already filled with objects that "steal" the clicks, right (so no need to add another one).

Reply:
I would expect the same, but as it seams, the objects steal only the left clicks. If you right click on any of the button you can move the scene, or open the popup context menu
![image](https://user-images.githubusercontent.com/19388042/83526469-c43d3380-a4e6-11ea-801a-7e316c0bd263.png)